cmd/helm/install.go

```diff
@@ -220,6 +220,12 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if client.ChartPathOptions.RepoURL != "" {
+		chartRequested.Metadata.RepoURL = client.ChartPathOptions.RepoURL
+	} else {
+		chartRequested.Metadata.RepoURL = "path"
```

Comment:
If the chart did not originate from a repository, recommend omitting inserting `path` into the `RepoURL` field

Reply:
This was discussed during the HIP process, and ultimately decided to use a placeholder string. I'm happy to do whatever here, but wanted to make sure I followed the original HIP - https://github.com/helm/community/blob/main/hips/hip-0018.md#specification

Reply:
I'd almost prefer the actual path. It's a good hint if I forgot where I installed from, maybe as a file:// url?

Reply:
local path it's nice feature, because in big teams we can see who touched chart at last time :)